### PR TITLE
divides accordion and accordion item readme

### PR DESCRIPTION
**Pull request by brianchristopherbrady**
Divides Accordion and Accordion Item docs into two separate README

### CODE REVIEW

File: packages/web-components/src/accordion-item/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+| `<AccordionItem>`   | `<fluent-accordion-item>` |
+| `<AccordionHeader>` | `named slot = "heading"`  |
+| `<AccordionPanel>`  | `default slotted content` |
+
+<br />
+
+**Property Mapping**
+| Fluent UI React 9 | Fluent Web Components 3 | Description of difference |
+|---------------------------|---------------------------|------------------------------------------------------------------------------------------|
+| `defaultOpenItems: number`| `expand: boolean` | _FuiR9_ `defaultOpenItems` is a number property set on the `Accordion` corresponding to the intended `AccordionItem` to be expanded.<hr /> `expand` is a boolean property set directly on the `AccordionItem` intended to be expanded.
```
Comment (chrisdholt) [inline packages/web-components/src/accordion-item/README.md:135]: Are these really parity? One looks like you enumerate an index to illustrate what should be open (fuir9) which would be akin to setting `expanded` on the item in WC.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/1ad2597f744bf7c3e2eada7720eea793fcee660e'>1ad2597f744bf7c3e2eada7720eea793fcee660e</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/accordion-item/README.md (hunk 1 above)
Comment (miroslavstastny) [inline packages/web-components/src/accordion-item/README.md:135]: I would expect `defaultOpenItems` to be documented both here in `accordion-item` and in `accordion` as well.

File: packages/web-components/src/accordion-item/README.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+| `multiple: boolean` | `expandmode: "single"` &#124; `"multiple"`| |
```
Comment (miroslavstastny) [inline packages/web-components/src/accordion-item/README.md:136]: This should be moved to `accordion`.
```suggestion
| `multiple: boolean` | `expand-mode: "single"` &#124; `"multiple"`| |
```

**Review by miroslavstastny — COMMENTED** [review]
I love the WC3 vs FUIR9 comparison!

**Review by chrisdholt — APPROVED** [review]

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**